lua/nvim-treesitter/install.lua

```diff
@@ -3,7 +3,7 @@ local fn = vim.fn
 local luv = vim.loop
 
 local M = {}
-local repositories = {
+M.repositories = {
```

Comment:
I was thinking, should we make a config.lua file where we put this kind of information ? Or maybe we could do like nvim-lsp where each language imports `config` and inserts its related data. It would be easier if we had maintainers for multiple languages.

Reply:
Oh that could be a great idea, I'll add a card for this in the roadmap.